.travis.yml

```diff
@@ -10,4 +10,3 @@ install:
 script:
   - go test -v -cover -covermode=count -coverprofile=cover.prof
   - cat cover.prof
-  - goveralls -coverprofile cover.prof -repotoken $COVERALLS_TOKEN
```

Comment:
removed this line because `$COVERALLS_TOKEN` is blank and I was getting this error `flag needs an argument: -repotoken`